apps/server/src/database/schemas/api-key.schema.ts

```diff
@@ -1,13 +1,13 @@
 import { Prop, Schema, SchemaFactory } from '@nestjs/mongoose';
+import { Document } from 'mongoose';
```

Comment:
Could you update to use `HydratedDocument` instead of `Document`, just to standardize this in the project?